### PR TITLE
Login signup Pages video listing

### PR DESCRIPTION
**Pull request by NamrataRaikwar2002**
- login signup page
- fetched data from db

csb: https://uizbwj.csb.app/

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,51 +1,60 @@
...
-.App-header {
-  background-color: #22201c;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  color: white;
+input {
+  outline: none;
```
Comment (GeetaPogul) [inline src/App.css:22]: remove this extra space

File: src/App.css [hunk 2]
```diff
@@ -1,51 +1,60 @@
...
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+body {
+  font-size: 1.6rem;
+  background-color: var(--bodyBg-color);
+  color: white;
```
Comment (GeetaPogul) [inline src/App.css:17]: use css color variable

File: src/Component/Navbar/Navbar.css [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+.icon_badge {
+    display: flex;
+    justify-self: center;
+    width: 1rem;
+    height: 1rem;
+    font-size: 0.7rem;
+    align-self: center;
+    font-weight: bolder;
+    color: white;
```
Comment (GeetaPogul) [inline src/Component/Navbar/Navbar.css:77]: here also

File: src/Component/Navbar/Navbar.jsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        <aside className="nav_rightside">
+          <Link to="/" className="navbar_link textForPrimaryColor productsText">
+            Explore
+          </Link>
+          <Link to='/login-page'>
+          <button className='btn card_btn btnHover'>
+            Login
+          </button>
+          </Link>
+          {/* will use this icon later */}
```
Comment (GeetaPogul) [inline src/Component/Navbar/Navbar.jsx:38]: remove this comment if you don't need this

**Review by GeetaPogul — APPROVED** [review]

File: src/Pages/VideoListing/Explore.jsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+                      videoLength={videoLength}
+                      thumbnail={thumbnail}
+                      channelName={channelName}
+                      channelProfile={channelProfile}
+                       />
+            )
+          })}
+          </div>
+      </main>
+      {/* <div> */}
```
Comment (manojasarsa) [inline src/Pages/VideoListing/Explore.jsx:41]: you can remove these comments if no longer needed

**Review by manojasarsa — APPROVED** [review]
Code looks clean. Just remove the comments if no longer needed

**Pull request merged**